core/mbed/source/interrupts.cpp

```diff
@@ -26,6 +26,11 @@ void vIRQ_SetVectorX(uint32_t irqn, uint32_t vector, uint32_t flag)
     }
 }
 
+void vIRQ_SetVector(uint32_t irqn, uint32_t vector)
```

Comment:
Please add this information to the commit message.

I guess this is a mainly a usability feature, as the knowledge to make the gateway with vIRQ_SetVectorX instead of vIRQ_SetVector is not widely known. In which case, after the "why" goes into your commit message, the commit seems good to me.